ci-operator/config/maistra/istio/maistra-istio-maistra-3.0.yaml

```diff
@@ -82,6 +96,245 @@ tests:
           memory: 100Mi
       timeout: 20m0s
     workflow: servicemesh-istio-e2e-aws
+- as: integ-unit-3-0
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 2h0m0s
+    version: "4.10"
+  steps:
+    env:
+      MAISTRA_BUILDER_IMAGE: registry.ci.openshift.org/ci/maistra-builder:upstream-master
+      MAISTRA_NAMESPACE: maistra-e2e-test
+      MAISTRA_SC_POD: maistra-e2e-test-sc-pod
+    test:
+    - as: copy-src
+      cli: latest
+      commands: |
+        # SRC_PATH does end with /. : the content of the source directory is copied into dest directory
+        oc cp ./. "${MAISTRA_NAMESPACE}"/"${MAISTRA_SC_POD}":/work/
+      env:
+      - name: MAISTRA_NAMESPACE
+      - name: MAISTRA_SC_POD
+      from: src-test
+      resources:
+        requests:
+          cpu: 100m
+          memory: 100Mi
+      timeout: 20m0s
+    - as: integ-unit-3-0
+      cli: latest
+      commands: |
+        oc rsh -n "${MAISTRA_NAMESPACE}" "${MAISTRA_SC_POD}" \
```

Comment:
You need these variables set somewhere: https://github.com/openshift/release/blob/35fa29ebd762c3273ed9fb68b7db1dc093603448/ci-operator/config/maistra/istio/maistra-istio-maistra-2.4.yaml#L28-L33

I'm not sure where the best place is. @yxun do you know?

Reply:
In workflow test, I think only need to set  BUILD_WITH_CONTAINER="0" . The other golang related env variables should be optional when running there.

If you want to set them, you can put them after the `make -e "T=-v -count=1" build racetest binaries-test"` so that they will in the same bash session.
In other way, you can also set them around the line 110
``` 
env:
      MAISTRA_BUILDER_IMAGE: registry.ci.openshift.org/ci/maistra-builder:upstream-master
      MAISTRA_NAMESPACE: maistra-e2e-test
      MAISTRA_SC_POD: maistra-e2e-test-sc-pod
```   
and then adding the env variable names around line 138
```
      env:
      - name: MAISTRA_NAMESPACE
      - name: MAISTRA_SC_POD
  ```